app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>NEW task</h1>
+
+<%= render 'form', task: @task %>
+
```

Comment:
Please stay just one empty line after end of code, and between code constructions